File: JS/ejercicio30.js
usuario="admin"
contrasena="1234"
rol="administrador"

if(usuario=="admin" && contrasena=="1234" && rol=="administrador"){
    console.log("bienvenido admin")
}
else if(usuario=="admin" && contrasena=="1234" && rol!="administrador"){
    console.log("bienvenido admin pero no tienes permisos de administrador")
}
else{
    console.log("usuario o contraseña incorrectos")
}

File: PYTHON/ejercicio30.py
usuario=input("ingrese su usuario:")
contrasena=input("ingrese su contraseña:")
rol=input("ingrese su rol:")

if(usuario=="admin" and contrasena=="1234" and rol=="administrador"):
    print("bienvenido admin")

elif(usuario=="admin" and contrasena=="1234" and rol!="administrador"):
    print("bienvenido admin pero no tienes permisos de administrador")

else:
    print("usuario o contraseña incorrectos")
